astro/src/content/blog/digital-twins.mdx

```diff
@@ -1,7 +1,8 @@
 ---
 publish_date: 2025-04-16
 title: "Digital Twins: Bridging Physical Assets and Digital Identities"
-description: Learn about digital twins—virtual replicas of physical systems—and their role in optimizing performance and reducing downtime in the real world. 
+seoTitle: "Digital Twin Technology and Identity | Applying Digital Twins Beyond Physical Assets"
```

Comment:
You want `htmlTitle` not `seoTitle`.

Reply:
`Applying Digital Twins Beyond Physical Assets` doesn't make much sense to me as a phrase.